examples/clustermanager/simple/UnixDomainCM.jl

```diff
@@ -9,7 +9,7 @@ function launch(manager::UnixDomainCM, params::Dict, launched::Array, c::Conditi
     for i in 1:manager.np
         sockname = tempname()
         try
-            cmd = `$(params[:exename]) --worker=custom $(@__FILE__) worker $sockname`
```

Comment:
@jakebolewski - sometime the workers were launched with "worker" option set as default (and this was responsible for #10318) . I suspect it to be due to the second "worker" argument towards the end. Do you think this is a possibility with the way `getopt_long` works?